features/google/src/model/entities/place-visit.js

```diff
@@ -0,0 +1,14 @@
+export default class PlaceVisit {
+    constructor({ timestamp, locationName }) {
+        this._timestamp = timestamp;
+        this._locationName = locationName;
+    }
+
+    get timestamp() {
```

Comment:
Can we maybe have an "Event" or "TimelineEvent" as base class so that we can deal with it and unify denomination?

Reply:
Yes, `TimelineEvent` would fit nicely.